app/javascript/components/auth-key-pair-cloud/auth-key-pair-cloud-form.schema.js

```diff
@@ -0,0 +1,35 @@
+import { componentTypes, validatorTypes } from '@@ddf';
+import { http } from '../../http_api';
+
+const providerOptions = () => http.get(`/auth_key_pair_cloud/ems_form_choices`).then((data) =>
```

Comment:
This should be queried through the API, but there's no support for that...I'll get back to you when it's ready.

Reply:
After these are in:
https://github.com/ManageIQ/manageiq/pull/20775
https://github.com/ManageIQ/manageiq-providers-amazon/pull/668
https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/114
https://github.com/ManageIQ/manageiq-providers-openstack/pull/662

You can use this:
```
API.get('/api/vms?expand=resources&attributes=id,name,supports_auth_key_pair_create&filter[]=supports_auth_key_pair_create=true')
```

Reply:
@kavyanekkalapu they are in, so you are good to go after checking out the latest core and a `bin/update`